test/integration/generated/report/AutoRestReport.xcodeproj/AutoRestReport_Info.plist

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<plist version="1.0">
+<dict>
+  <key>CFBundleDevelopmentRegion</key>
+  <string>en</string>
+  <key>CFBundleExecutable</key>
+  <string>$(EXECUTABLE_NAME)</string>
+  <key>CFBundleIdentifier</key>
```

Comment:
So AutoRestReport is a generated library with test cases?

Reply:
Yes, the test case is to retrieve the report.